t/22-svirt.t

```diff
@@ -1034,4 +1034,126 @@ subtest 'get_wait_still_screen_on_here_doc_input' => sub {
     is($svirt->get_wait_still_screen_on_here_doc_input({}), 0, 'wait_still_screen on here doc is not set for kvm');
 };
 
+subtest 'Test routine consoles::sshVirtsh::provide_image_vmware_in_ds' => sub {
+    # Base subdir of this test; real default is /vmfs/volumes.
+    # After test completed, all contents will be deleted.
+    my $my_test_basedir = tempdir($dir . '/tb_XXXX');
+    my $nfs_ds = 'openqa_test';
+    my $ds = 'datastore_test';
+    set_var(VIRSH_OPENQA_BASEDIR => $my_test_basedir);
+    set_var(VMWARE_DATASTORE => $ds);
+    set_var(VMWARE_NFS_DATASTORE => $nfs_ds);
+    set_var(VMWARE_NFS_DATASTORE_DEBUG => 0);
```

Comment:
Don't use testapi methods in this backend unit test

Reply:
Looking from top in this test module, that test api `set_var` was already used before my new test, therefore I used it too. I could change it, but in this case all the other occurrences should too.

Reply:
Does it still make sense skipping testapi here only?

Reply:
Yes, please. You don't need to change existing code but for new code just follow this approach